ballerina/sql_client.bal

```diff
@@ -67,6 +67,11 @@ public client class SQLClient {
         return resultStream;
     }
 
+    public function runExecuteQuery(sql:ParameterizedQuery filterClause, typedesc<record {}> t) returns stream<record {}, sql:Error?>|error {
```

Comment:
```suggestion
    public function runExecuteQuery(sql:ParameterizedQuery filterClause, typedesc<record {}> rowType) returns stream<record {}, sql:Error?>|error {
```

Reply:
rename input param

Reply:
Fixed in [33dc252](https://github.com/ballerina-platform/module-ballerina-persist/pull/14/commits/33dc252c4b4316ad45d18be287633ade5156637d)